### PR TITLE
SECURITY.md

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### What does this implement/fix? Explain your changes.
- `SECURITY.md` added

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+| 6.1           | :white_check_mark: |
+| < 6.1         | :x:                |
+
+## Reporting a vulnerability
+
+Please report security vulnerabilities by email to [info@ascii-art.site](mailto:info@ascii-art.site "info@ascii-art.site").
+
+If the security vulnerability is accepted, a patch will be crafted privately
+in order to prepare a dedicated bugfix release as soon as possible (depending
+on the complexity of the fix).
```
Comment (sadrasabouri) [inline SECURITY.md:16]: Why privately? I know we are using the scikit-learn template. But it does not make sense to me.

File: otherfile/version_check.py [hunk 2]
```diff
@@ -29,11 +29,13 @@
 PARAMS_ITEMS = ['ART_VERSION = "{0}"']
 META_ITEMS = ['% set version = "{0}" %']
 ISSUE_TEMPLATE_ITEMS = ["- Art {0}"]
+SECURITY_ITEMS = ["| {0}           | :white_check_mark: |", "| < {0}         | :x:                |"]
 
 FILES = {
     "setup.py": SETUP_ITEMS,
     "INSTALL.md": INSTALL_ITEMS,
     "CHANGELOG.md": CHANGELOG_ITEMS,
+    "SECURITY.md": SECURITY_ITEMS,
```
Comment (sadrasabouri) [inline otherfile/version_check.py:38]: I think we don't need to update this each time.
I know that we are using the scikit-learn template but it's wired to update the security-supported version each time after a release.

**Review by sadrasabouri — APPROVED** [review]

**Review by sadrasabouri — CHANGES_REQUESTED** [review]
I just raised some concerns. Hope they are useful.

File: otherfile/version_check.py (hunk 2 above)
Comment (sepandhaghighi) [reply otherfile/version_check.py:38]: Here we have two options:

1. Support only one version (the latest stable one)
2. Support a range of versions

* In the second option if someone finds a vulnerability in a supported version, which is not the latest, we should release a specific bugfix for that version. You can refer to how different Python versions are maintained.

File: SECURITY.md (hunk 1 above)
Comment (sepandhaghighi) [reply SECURITY.md]: Good point.
fixed in https://github.com/sepandhaghighi/art/pull/269/commits/55a01836f52a30c28d658e2d3ba53563c4b68281

File: otherfile/version_check.py (hunk 2 above)
Comment (sepandhaghighi) [reply otherfile/version_check.py:38]: If you disagree with the correct option, we can discuss it further.

**Comment by sepandhaghighi** [comment]
> I just raised some concerns. Hope they are useful.

We are currently planning to develop a security template and would appreciate any suggestions or ideas you may have. Please feel free to share your thoughts with us.

**Review by sadrasabouri — APPROVED** [review]

**Pull request merged**